src/server/raftkv.rs

```diff
@@ -107,6 +110,7 @@ impl From<RaftServerError> for KvError {
 pub struct RaftKv<S: RaftStoreRouter<RocksSnapshot> + 'static> {
     router: S,
     engine: RocksEngine,
+    region: Arc<Region>,
```

Comment:
Is it necessary?

Reply:
Yes. I think we shall not clone the whole structure for every request which acquire a `RegionSnapshot`.